Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -494,6 +494,10 @@ theorem eq_symm_comp {α β γ} (e : α ≃ β) (f : γ → α) (g : γ → β)
 theorem symm_comp_eq {α β γ} (e : α ≃ β) (f : γ → α) (g : γ → β) : e.symm ∘ g = f ↔ g = e ∘ f :=
   ((Equiv.refl γ).arrowCongr e).symm_apply_eq
 
+theorem trans_eq_refl_iff_eq_symm (f : α ≃ β) (g : β ≃ α) :
```

Comment:
We have four variants of this theorem for `mul_eq_one`, so probably we should have the same thing for this lemma.

Can you also make the `f` and `g` arguments implicit to match?